### PR TITLE
Implemented Serialization Helper for GoogleSheets Converter.

### PR DESCRIPTION
**Pull request by pawellysy**
Implemented Serializer helper for google sheets converter.

### CODE REVIEW

File: ts/Dashboard/SerializeHelper/GoogleSheetsConverterHelper.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    /* *
+     *
+     *  Declarations
+     *
+     * */
+
+    export interface JSON extends Serializable.JSON<'Data.GoogleSheetsConverter'>{
+        options: GoogleSheetsConverter.Options;
+        parseDateFn?: string;
```
Comment (bre1470) [inline ts/Dashboard/SerializeHelper/GoogleSheetsConverterHelper.ts:82]: This is not implemented and should be removed.

File: ts/Dashboard/SerializeHelper/GoogleSheetsConverterHelper.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ *
+ *  Functions
+ *
+ * */
+/**
+ * Change the obj of GoogleSheetsConverter its Serialized form.
+ * @param obj Object to serialize
+ * @return Serialized object
+ */
+
```
Comment (bre1470) [inline ts/Dashboard/SerializeHelper/GoogleSheetsConverterHelper.ts:37]: Spacing should be corrected. Same over import section comment.

**Review by bre1470 — CHANGES_REQUESTED** [review]

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

File: ts/Dashboard/SerializeHelper/GoogleSheetsConverterHelper.ts [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+/* *
+ *
+ *  Import
+ *
+ * */
+
+import Serializable from '../Serializable.js';
+import U from '../../Core/Utilities.js';
+import GoogleSheetsConverter from '../../Data/Converters/GoogleSheetsConverter.js';
+const { merge } = U;
```
Comment (karolkolodziej) [inline ts/Dashboard/SerializeHelper/GoogleSheetsConverterHelper.ts:26]: Move declaration under the direct import.

**Review by karolkolodziej — APPROVED** [review]

**Pull request merged**